code/obj/artifacts/artifact_items/talisman.dm

```diff
@@ -0,0 +1,208 @@
+#define SWIFTNESS 1
+#define FORTUNE 2
+#define SPACEFARING 3
+#define ELEMENTS 4
+#define STRENGTH 5
+#define PROTECTION 6
+
+/obj/item/artifact/talisman
+	name = "artifact talisman"
+	icon = 'icons/obj/artifacts/artifactsitemS.dmi'
+	associated_datum = /datum/artifact/talisman
+	w_class = W_CLASS_SMALL
+	var/associated_effect
+	var/mob/living/carbon/human/active_user
+	/// talisman fault/glimmer effect tied to this artifact
+	var/datum/statusEffect/talisman_held/talisman_effect
+
+	// swiftness vars
+	var/swiftness_mod = 0
+	// fortune vars
+	var/datum/statusEffect/talisman_fortune/fortune_effect
+	// spacefaring vars
+	var/space_prot = 0
+	// elements vars
+	var/heat_prot = 0
+	var/cold_prot = 0
+	// strength vars
+	var/extra_hp = 0
```

Comment:
Couldn't these all be collapsed into a single var? The type already differentiates how the var used.

Reply:
Um if you think it should be done. I don't see much of a benefit in doing so, I just see the var name being made less clear and for where multiple values are chosen it would be a list

Reply:
Definitely doesn't need to be done. Just an observation that a variable indicating the magnitude of the change would cover the majority of the use since only one type can be active at a time aside from the one protection outlier.